src/org/usfirst/frc/team2399/robot/RobotMap.java

```diff
@@ -128,5 +125,4 @@
 	// TODO: Figure out what this time should actually be
 	// Created a constant for how long the solenoid should run
 	public static final double SHIFT_TIMER = 0.5;
```

Comment:
I'd like to CR the shifter code/suggest some changes if this number reflects something about how it works. /endsoapbox